src/app/plan/components/Wizard/WizardContainer.tsx

```diff
@@ -13,6 +13,7 @@ export interface IFormValues {
   selectedStorage: string;
   selectedNamespaces: any[];
   persistentVolumes: any[]; // TODO replace this with selections-only version after refactor
+  pvStorageClassAssignment: any[];
```

Comment:
Let's try to avoid using `any` when we can (we should eventually make better types for all of these `any` props). In this case it looks like `pvStorageClassAssignment` is an object with arbitrary string keys that map to string values.
```
pvStorageClassAssignment: {
  [key: string]: string;
}
```

Reply:
Also I hadn't noticed the `[]` on `any[]`. I think it's an object, not an array

Reply:
This doesn't work.
Although I agree with the target to get rid of `any` it seems there is still lots of work to get there.
All the properties above are any[] too.
I believe it should be addressed separately.

Reply:
I think the type errors you were getting were because the `mapStateToProps` function that initializes the values was also passing in `[]` instead of `{}`. I think I may address this in my next PR for the storage class table anyway though.